### PR TITLE
Senate page

### PR DESCRIPTION
**Pull request by gopperman**

### CODE REVIEW

File: common/containers/Election.js [hunk 1]
```diff
@@ -139,7 +139,7 @@ class Election extends Component {
 								<span>Featured Races</span>
 							</h3>
 							<div className='r-row--full'>
-								<BalanceOfPower {...bopData} />
+								<BalanceOfPower {...bopData} displayLink={true} />
```
Comment (gabrielflorit) [inline common/containers/Election.js]: No need to set a boolean attribute to true - but linting should take care of that. Did you lint?

File: common/containers/Senate.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+					overline={nameUtil.race.name(race)}
+					precinctsReportingPct={stateUnit.precinctsReportingPct}
+					candidates={sortByVoteCount(candidates)}
+					buttonText='See full results'
+					buttonUrl={urlManager.race(race)} />
+			)
+
+		})
+
+		const heroClass = compareStringsNoAlpha(firstRace.statePostal, 'ma') ?
```
Comment (gabrielflorit) [inline common/containers/Senate.js]: Is this necessary?

File: common/routes.js [hunk 3]
```diff
@@ -28,6 +29,10 @@ export default (
 			component={PresidentMA} />
 
 		<Route
+			path={`${urlManager.base()}/US%2520Senate`}
```
Comment (gabrielflorit) [inline common/routes.js]: Best to set the path with urlManager - that way we let it handle all urls.

So something like
```
path={urlManager.office({ officeName: 'US House' })}
```

File: common/utils/visUtils.js [hunk 4]
```diff
@@ -3,6 +3,49 @@
...
+			holdovers: 34,
+			won: _.get(trends, 'dem', 0),
+		},
+		gop: {
+			holdovers: 30,
+			won: _.get(trends, 'gop', 0),
+		},
+		ind: {
+			holdovers: 2,
+			won: _.get(trends, 'ind', 0),
```
Comment (gabrielflorit) [inline common/utils/visUtils.js]: This assumes that we will only have candidates of party `dem|gop|ind`, which is not a correct assumption. Use `standardize.normalizeParty` to normalize all independents to `ind`.

File: common/routes.js (hunk 3 above)
Comment (gabrielflorit) [inline common/routes.js]: Also this reminds me that `routes.js` should use `urlManager` for all routes - will add an issue.

File: common/containers/Election.js (hunk 1 above)
Comment (gopperman) [inline common/containers/Election.js]: done

File: common/routes.js (hunk 3 above)
Comment (gopperman) [inline common/routes.js]: done

File: common/utils/visUtils.js (hunk 4 above)
Comment (gopperman) [inline common/utils/visUtils.js]: made the anonymous function above use that function....done

**Pull request merged**